action.yaml

```diff
@@ -51,6 +51,9 @@ inputs:
     description: 'Whether to create a Port entity for the repository'
     required: false
     default: "true"
+outputs:
+  repoURL:
```

Comment:
I would rename this to camel case `repo-url` that way it follows a bit more what is commonly used in our code base such as here:
* https://github.com/SonarSource/gh-action_releasability/blob/master/action.yml#L5

And more commonly also used by GitHub:
https://github.com/SonarSource/gh-action_release/blob/master/.github/workflows/release.yml#L32

Reply:
I agree but I couldn't find a common pattern, and the purpose of camelCase here was for consistency with the inputs.

In port-actions, we use underscore, dash in releasability and sbom, and a mix of camelCase, dash and underscore in release action.

Reply:
ok so we keep doing a mix, ack.